MyNotes/Form1.cs

```diff
@@ -1,42 +1,34 @@
 ﻿using System;
```

Comment:
No need to keep unused "using" - visual studio/resharper/rider will allow you to clean it automatically.